csunplugged/classic/models.py

```diff
@@ -0,0 +1,36 @@
+"""Models for the classic application."""
+
+from django.db import models
+
+
+class ClassicPage(models.Model):
+    """Model for Classic CS Unplugged page in database."""
+
+    #  Auto-incrementing 'id' field is automatically set by Django
+    slug = models.SlugField(unique=True)
+    name = models.CharField(max_length=200)
+    redirect = models.URLField()
+
+    def get_absolute_url(self):
+        """Return the canonical URL for a ClassicPage.
+
+        Returns:
+            URL as string.
+        """
+        return self.redirect
+
+    def model_type(self):
```

Comment:
I noticed this is in all the models, remind me where this is used besides in the tests?

Reply:
It's the label displayed on the search page.

Reply:
OH that's why it's there :+1: